src/cfnlint/customRules/Operators.py

```diff
@@ -0,0 +1,13 @@
+import cfnlint.custom_rules
```

Comment:
just need that legal boilerplate here as well
```suggestion
"""
Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
SPDX-License-Identifier: MIT-0
"""
import cfnlint.custom_rules
```